sdk/core/azure-core/inc/azure/core/azure_assert.hpp

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+/**
+ * @file
+ * @brief Deprecated assert macros.
+ *
+ */
+
+#include "azure/core/internal/azure_assert.hpp"
+
+#define AZURE_ASSERT(exp) _azure_ASSERT(exp)
+#define AZURE_ASSERT_MSG(exp, msg) _azure_ASSERT_MSG(exp, msg)
+#define AZURE_ASSERT_FALSE(exp) _azure_ASSERT_FALSE(exp)
+#define AZURE_UNREACHABLE_CODE() _azure_UNREACHABLE_CODE()
+#define AZURE_NOT_IMPLEMENTED() _azure_NOT_IMPLEMENTED()
```

Comment:
Are all of these used within upstream consumers like storage? If some aren't, we could remove those.

Reply:
I checked prior public non-beta releases, all of them were used.